src/plonkish/compiler/mod.rs

```diff
@@ -525,13 +512,10 @@ fn add_q_last_to_constraint<F: Field>(
     constraint: PolyExpr<F>,
 ) -> PolyExpr<F> {
     let q_last_column = unit.last_step.clone().expect("last column not found").1;
-    let q_last = PolyExpr::<F>::Query(q_last_column, 0, "q_last".to_owned());
-    let not_q_last_expr = PolyExpr::Sum(vec![
-        PolyExpr::Const(F::ONE),
-        PolyExpr::Neg(Box::new(q_last)),
-    ]);
+    let q_last = q_last_column.query(0, "q_last".to_owned());
+    let not_q_last_expr = PolyExpr::Const(F::ONE) + (-q_last);
 
```

Comment:
Saw this construction a few times, but what's the difference between -q-last and + (-q_last)?

Reply:
None, just for clarity and avoiding problems with the operator overloading